honeycomb/honeycomb.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2019, Honeycomb, Hound Technology, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package honeycomb contains a trace exporter for Honeycomb
+package honeycomb
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"time"
+
+	libhoney "github.com/honeycombio/libhoney-go"
+	"go.opentelemetry.io/sdk/trace"
+)
+
+// Exporter is an implementation of trace.Exporter that uploads a span to Honeycomb
+type Exporter struct {
+	Builder        *libhoney.Builder
+	SampleFraction float64
```

Comment:
it's not clear if this is interpreted as N or 1/N...

Reply:
Good point. I'm not sure how OpenTelemetry passes up the sample rate yet. In OpenCensus, they set it as a fraction (1/N). We then translate that to SampleRate (N).
I think I'm going to comment this part out for now and make a task to investigate how OpenTelemetry will surface sampling to the user/exporters.
https://app.asana.com/0/1136461855218693/1139292285891801?du=579988137396441